src/main/java/com/mindera/school/spaceshiprent/properties/ThirdPartyProperties.java

```diff
@@ -0,0 +1,34 @@
+package com.mindera.school.spaceshiprent.properties;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+@ConfigurationProperties(prefix = "app.third-party")
+public class ThirdPartyProperties {
+
+    private Mailjet mailjet;
+
+    public static class Mailjet {
```

Comment:
The name of this object should be more generic, as it can be used by any third party you may add in the future. `Keys` for example could be a good name